update/factory/fullSyncInterceptors.go

```diff
@@ -301,7 +296,8 @@ func (ficf *fullSyncInterceptorsContainerFactory) generateShardHeaderInterceptor
 			continue
 		}
 
-		interceptor, err := ficf.createOneShardHeaderInterceptor(identifierHeader)
+		var interceptor process.Interceptor
```

Comment:
You have done this only for err shadowing warning?

Reply:
yes, reverted, renamed err var to errCreate